ethash/src/progpow.rs

```diff
@@ -14,6 +14,28 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+//! ProgPoW (Progressive Proof-of-Work) is the Ethereum network's proposed new Application-Specific Integrated
+//! Circuit (ASIC) resistant Proof-of-Work mining algorithm.
+//!
+//! ProgPoW's aim is to reduce the efficiencies of specialized mining devices known as ASIC chips
+//! (and accelerated GPU-based setups), and to maximize the performance of General Purpose Hardware (GPUs) to enable
+//! more users to compete for new cryptocurrency awarded by the protocol.
+//!
+//! ASIC chips are those that are tailored to efficiently mining cryptocurrency based on a specific hashing algorithm.
+//!
+//! GPU mining setups are less specialised are struggle to compete for mining rewards.
+//!
+//! It would be a change from Ethereum's current PoW mining algorithm known as Ethash and would impact the
+//! Ethereum network's mining rewards.
+//!
+//! ProgPoW has been trialed on client implementations running the "Gangnam" testnet.
+//!
```

Comment:
I don't know whether the last four lines below are relevant here 😅

Reply:
I've pushed a commit removing the unnecessary lines